packages/shared/forks/ReactFeatureFlags.native-oss.js

```diff
@@ -10,100 +10,105 @@
 import typeof * as FeatureFlagsType from 'shared/ReactFeatureFlags';
 import typeof * as ExportsType from './ReactFeatureFlags.native-oss';
 
+// TODO: Align these flags with canary and delete this file once RN ships from Canary.
+
 // -----------------------------------------------------------------------------
-// Ready for next major.
+// TODO for next React Native major.
 //
-// Alias __NEXT_MAJOR__ to false for easier skimming.
+// Alias __TODO_NEXT_RN_MAJOR__ to false for easier skimming.
 // -----------------------------------------------------------------------------
-const __NEXT_MAJOR__ = false;
+const __TODO_NEXT_RN_MAJOR__ = false;
+export const enableCache = __TODO_NEXT_RN_MAJOR__;
+export const enableRefAsProp = __TODO_NEXT_RN_MAJOR__;
+export const disableStringRefs = __TODO_NEXT_RN_MAJOR__;
+export const disableLegacyMode = __TODO_NEXT_RN_MAJOR__;
+export const enableBigIntSupport = __TODO_NEXT_RN_MAJOR__;
+export const useModernStrictMode = __TODO_NEXT_RN_MAJOR__;
+export const enableReactTestRendererWarning = __TODO_NEXT_RN_MAJOR__;
+export const enableAsyncActions = __TODO_NEXT_RN_MAJOR__;
+export const consoleManagedByDevToolsDuringStrictMode = __TODO_NEXT_RN_MAJOR__;
+export const enableDeferRootSchedulingToMicrotask = __TODO_NEXT_RN_MAJOR__;
+export const alwaysThrottleDisappearingFallbacks = __TODO_NEXT_RN_MAJOR__;
+export const alwaysThrottleRetries = __TODO_NEXT_RN_MAJOR__;
+export const enableInfiniteRenderLoopDetection = __TODO_NEXT_RN_MAJOR__;
+export const enableRenderableContext = __TODO_NEXT_RN_MAJOR__;
+export const enableComponentStackLocations = __TODO_NEXT_RN_MAJOR__;
+export const disableModulePatternComponents = __TODO_NEXT_RN_MAJOR__;
 
+// -----------------------------------------------------------------------------
+// Enabled for next React Native major.
+// Hardcode these to true after the next RN major.
+//
+// Alias __NEXT_RN_MAJOR__ to true for easier skimming.
+// -----------------------------------------------------------------------------
+const __NEXT_RN_MAJOR__ = true;
+export const disableClientCache = __NEXT_RN_MAJOR__;
+export const disableLegacyContext = __NEXT_RN_MAJOR__;
+export const enableCacheElement = __NEXT_RN_MAJOR__;
+export const enableTaint = __NEXT_RN_MAJOR__;
+export const enableUnifiedSyncLane = __NEXT_RN_MAJOR__;
+export const enableFizzExternalRuntime = __NEXT_RN_MAJOR__; // DOM-only
+export const disableJavaScriptURLs = __NEXT_RN_MAJOR__; // DOM-only
+export const enableFormActions = __NEXT_RN_MAJOR__; // DOM-only
+export const enableBinaryFlight = __NEXT_RN_MAJOR__; // DOM-only
+export const enableCustomElementPropertySupport = __NEXT_RN_MAJOR__; // DOM-only
+export const enableServerComponentKeys = __NEXT_RN_MAJOR__;
+export const enableServerComponentLogs = __NEXT_RN_MAJOR__;
```

Comment:
@yungsters @rubennorte some of these likely went out in 0.74 right?

Reply:
I know that `enableCustomElementPropertySupport` was enabled after the branch cut for 0.74, but I'm not sure about the other three.